lib/bot.js

```diff
@@ -300,6 +300,22 @@ module.exports = (function() {
 			});
 		},
 
+		getPagesTranscludedBy: function (template, callback) {
```

Comment:
So shouldn't this function be called a bit different, let's say `getPagesWithTemplate`?